components/inputs/input-label-styles.js

```diff
@@ -27,6 +26,13 @@ export const inputLabelStyles = css`
 		left: auto;
 		right: 0.15rem;
 	}
+	:host([skeleton]) .d2l-input-label.d2l-skeletize::before {
+		bottom: 0.25rem;
+		top: 0.15rem;
+	}
+	:host([skeleton][required]) .d2l-input-label.d2l-skeletize::after {
+		display: none;
+	}
```

Comment:
This hides the "required" asterix when in skeleton-mode, which is what design would like.